### PR TITLE
Javascript Question Added

### PR DESCRIPTION
**Pull request by shubhankar-shandilya-india**
I have added a Javascript Question related to the scope used in javascript.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

I have added a question on the scope used in Javascript.
I have written its explanation and linked an freeCodeCamp article

### CODE REVIEW

**Comment by jdwilkin4** [comment]
Please make sure not to add `Closes #14 ` to your PR's because that issue is supposed to stay open for multiple contributors.

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,15 @@
 const javascriptQuiz = [
+  {
+    Question:
+      "Which of the following scoping type does JavaScript use?",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:4]:
```suggestion
      "Which of the following scoping types does JavaScript use?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by shubhankar-shandilya-india** [comment]
I have made the required changes.

**Comment by shubhankar-shandilya-india** [comment]
If all changes done please approve and merge my Pull request.
Thankyou.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by jdwilkin4** [comment]
After further review, it looks like this question has come from this quiz site
https://www.sanfoundry.com/javascript-mcqs-closures/

As such, we are removing this question from our site because we don't endorse plagiarism.

**Pull request merged**